augur/tasks/github/facade_github/tasks.py

```diff
@@ -51,14 +51,14 @@ def process_commit_metadata(session,contributorQueue,repo_id):
         login = None
     
         #Check the contributors table for a login for the given name
-        try:
-            query = session.query(Contributor).filter_by(cntrb_full_name=name)
-            contributors_with_matching_name = execute_session_query(query, 'one')
 
-            login = contributors_with_matching_name.gh_login
+        query = session.query(Contributor).filter_by(cntrb_full_name=name)
+        contributors_with_matching_name = execute_session_query(query, 'first')
 
-        except NoResultFound as e:
-            session.logger.debug(f"Failed local login lookup with error: {e}")
+        if not contributors_with_matching_name:
```

Comment:
Will taking this out of a try except cause the collection to fail in the case that a person is not identified?

Reply:
No, because it uses `first` which will return None if it doesn't exist. Where `one` expects a row to exist